src/utils-lgpl/fcplay.c

```diff
@@ -378,6 +385,33 @@ void play_samples(char **files, unsigned int card, unsigned int device,
 			file = fopen(name, "rb");
 			if (!file)
 				goto TRACK_EXIT;
+
+			if (verbose)
+				printf("Playing file %s On Card %u device %u, with buffer of %lu bytes\n",
+					    name, card, device, buffer_size);
+
+			if (gapless) {
+				parse_file(name, &codec);
+
+				rc = compress_next_track(compress);
+				if (rc)
+					fprintf(stderr, "ERR: compress next track set\n");
+
+				mdata.encoder_delay = 2048;
+				mdata.encoder_padding = 1024;
```

Comment:
why are we hardcoding this?

Reply:
this changes is not required now, will drop this change.

thanks,
mohs.